dotcom-rendering/src/components/ManyNewsletterSignUp.importable.tsx

```diff
@@ -204,23 +177,73 @@ export const ManyNewsletterSignUp = ({ apiEndpoint }: Props) => {
 		};
 	}, [toggleNewsletter, newslettersToSignUpFor]);
 
+	const sendRequest = async (reCaptchaToken: string): Promise<void> => {
+		reportTrackingEvent('ManyNewsletterSignUp', 'form-submit', {
+			newsletterIds: newslettersToSignUpFor,
+		});
+
+		const response = await requestMultipleSignUps(
+			email,
+			newslettersToSignUpFor,
+			reCaptchaToken,
+		);
+
+		if (!response.ok) {
+			const responseText = await response.text();
+			reportTrackingEvent('ManyNewsletterSignUp', 'failure-response', {
+				newsletterIds: newslettersToSignUpFor,
+				responseText,
+			});
+
+			return setStatus('Failed');
+		}
+
+		reportTrackingEvent('ManyNewsletterSignUp', 'success-response', {
+			newsletterIds: newslettersToSignUpFor,
+		});
+		setStatus('Success');
+	};
+
 	const handleSubmitButton = async () => {
 		if (status !== 'NotSent') {
 			return;
 		}
-		setStatus('Loading');
 
-		const result = await sendSignUpRequest(
-			email,
-			newslettersToSignUpFor,
-			apiEndpoint,
-		);
+		if (!useReCaptcha) {
+			setStatus('Loading');
+			void sendRequest('');
+			return;
+		}
```

Comment:
This was mostly a necessity to make the story work - but with the change to the config in "preview.js", we don't need to have a prop for this - it can check the switch value for `emailSignupRecaptcha` to determine whether to render and execute the Recaptcha component or send the request without a Recaptcha token string. "preview.js" sets `emailSignupRecaptcha` to false, so in the story, the Recaptcha won't be used.

The same switch is checked in the `buildNewsletterSignUpFormData` function - if false, the token is not included in the posted data.

This seems better now - if the config was changed in frontend to turn the off `emailSignupRecaptcha`, DCR would neither collect the token or try to include one in the post and (as long as  frontend had also changed the route handle to not require the token) the component would still work.